/test pull-kubernetes-verify-govet-levee
Unsure what caused the panic but running `hack/verify-govet-levee.sh` hasn't shown any issues locally.

```
 # crypto/elliptic
panic: no concrete method: func (*crypto/elliptic.nistCurve[*crypto/internal/nistec.P224Point]).Add(x1 *math/big.Int, y1 *math/big.Int, x2 *math/big.Int, y2 *math/big.Int) (*math/big.Int, *math/big.Int)
```
```
 # go/token
panic: no concrete method: func (*sync/atomic.Pointer[go/token.File]).CompareAndSwap(old *go/token.File, new *go/token.File) (swapped bool)
```

--- a/test/conformance/testdata/conformance.yaml
+++ b/test/conformance/testdata/conformance.yaml
@@ -594,6 +594,22 @@
     and minor versions MUST only be an integer.
   release: v1.19
   file: test/e2e/apimachinery/server_version.go
+- testname: ControllerRevision, resource lifecycle
+  codename: '[sig-apps] ControllerRevision [Serial] should manage the lifecycle of
+    a ControllerRevision [Conformance]'
+  description: Creating a DaemonSet MUST succeed. Listing all ControllerRevisions
+    with a label selector MUST find only one. After patching the ControllerRevision
+    with a new label, the label MUST be found. Creating a new ControllerRevision for
+    the DaemonSet MUST succeed. Listing the ControllerRevisions by label selector
+    MUST find only two. Deleting a ControllerRevision MUST succeed. Listing the ControllerRevisions
+    by label selector MUST find only one. After updating the ControllerRevision with
+    a new label, the label MUST be found. Patching the DaemonSet MUST succeed. Listing
+    the ControllerRevisions by label selector MUST find only two. Deleting a collection
+    of ControllerRevision via a label selector MUST succeed. Listing the ControllerRevisions
+    by label selector MUST find only one. The current ControllerRevision revision
+    MUST be 3.
+  release: v1.25
+  file: test/e2e/apps/controller_revision.go
 - testname: CronJob Suspend
   codename: '[sig-apps] CronJob should not schedule jobs when suspended [Slow] [Conformance]'
   description: CronJob MUST support suspension, which suppresses creation of new jobs.

--- a/test/e2e/apps/controller_revision.go
+++ b/test/e2e/apps/controller_revision.go
@@ -104,7 +104,24 @@ var _ = SIGDescribe("ControllerRevision [Serial]", func() {
 		framework.ExpectNoError(err)
 	})
 
-	ginkgo.It("should manage the lifecycle of a ControllerRevision", func() {
+	/*
+		Release: v1.25
+		Testname: ControllerRevision, resource lifecycle
+		Description: Creating a DaemonSet MUST succeed. Listing all
+		ControllerRevisions with a label selector MUST find only one.
+		After patching the ControllerRevision with a new label, the label
+		MUST be found. Creating a new ControllerRevision for the DaemonSet
+		MUST succeed. Listing the ControllerRevisions by label selector
+		MUST find only two. Deleting a ControllerRevision MUST succeed.
+		Listing the ControllerRevisions by label selector MUST find only
+		one. After updating the ControllerRevision with a new label, the
+		label MUST be found. Patching the DaemonSet MUST succeed. Listing the
+		ControllerRevisions by label selector MUST find only two. Deleting
+		a collection of ControllerRevision via a label selector MUST succeed.
+		Listing the ControllerRevisions by label selector MUST find only one.
+		The current ControllerRevision revision MUST be 3.
+	*/
+	framework.ConformanceIt("should manage the lifecycle of a ControllerRevision", func() {
 		csAppsV1 := f.ClientSet.AppsV1()
 
 		dsLabel := map[string]string{"daemonset-name": dsName}

